pkg/k8s/mocks/mocks.go

```diff
@@ -513,6 +513,15 @@ func (m *MockPipelineRun) UpdateResult(arg0 v1alpha1.Result) error {
 	return ret0
 }
 
+// UpdateResult mocks base method
+func (m *MockPipelineRun) UpdateResultNoPush(arg0 v1alpha1.Result) error {
```

Comment:
I added this manually. Of course such a change should be applied by the responsible code generator.

Reply:
using generator in the meanwile (`hack/update-codegen.sh --gen-mocks`)